agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/interfaces/GeneModelInterface.java

```diff
@@ -0,0 +1,23 @@
+package org.alliancegenome.indexer.indexers.curation.interfaces;
+
+import com.fasterxml.jackson.annotation.JsonView;
+import jakarta.ws.rs.*;
+import jakarta.ws.rs.core.MediaType;
+import org.alliancegenome.curation_api.model.document.es.AffectedGenomicModelDocument;
+import org.alliancegenome.curation_api.response.SearchResponse;
+import org.alliancegenome.curation_api.view.View;
+
+import java.util.HashMap;
+
+@Path("/model")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public interface GeneModelInterface {
```

Comment:
There is a GeneModelInterface in curation with the same exact definition. That one should be used and this one deleted.

Reply:
That class is not used any longer. I'll delete it.